Update Kiwi building documentation

aaannz opened the pull request:
- RPM certificate package info
- accepted sources format and examples
- source example
- troubleshooting and limitations
- new screenshot for profile creation

mbologna commented on adoc/advanced_topics_image_management.adoc:
```diff
@@ -405,9 +422,34 @@ From the {susemgr}{webui} perform the following steps to configure a build host.
+====
 
 [[at.images.kiwi.buildchannels]]
-==== Define Container Build Channels with an Activation Key
```
Please state in a NOTE that an AK is mandatory for OS image builds

mbologna commented on adoc/advanced_topics_image_management.adoc:
```diff
@@ -474,11 +557,63 @@ Kiwi source consists of at least `config.xml` file, usually `config.sh` and `ima
+[NOTE]
 ====
-!!!!!!!!!!!! add link to published JeOS sources
+{suse} provides fex examples of fully functional image sources at https://github.com/SUSE/manager-build-profiles[SUSE/manager-build-profiles] public GitHub repository.
```
fex?

mbologna commented on adoc/advanced_topics_image_management.adoc:
```diff
@@ -474,11 +557,63 @@ Kiwi source consists of at least `config.xml` file, usually `config.sh` and `ima
+    <packages type="bootstrap">
+        ...
+        <package name="sles-release"/>
+        <package name="rhn-org-trusted-ssl-certosimage" bootinclude="true"/>
```
As an additional repeat for the end user, please also add a comment here saying that this is required

mbologna commented on adoc/advanced_topics_image_management.adoc:
```diff
@@ -511,9 +646,22 @@ When you have selected a build profile detailed information about the selected p
 The following are some known pitfalls when working with images.
 
-* HTTPS certificates to access the registry or the git repositories should be deployed to the minion by a custom state file.
+* HTTPS certificates to access the HTTP sources or Git repositories should be deployed to the minion by a custom state file.
```
or configured directly on the minion by the end user

aaannz commented:
Addressed all comments + run spellcheck on advanced topics file.

mbologna commented:
Please make sure to address the explanation of the certificate generation:
you can see some details here: https://github.com/SUSE/spacewalk/blob/Manager-Kiwi-OS-image-building/spacewalk/certs-tools/mgr-package-rpm-certificate-osimage.sgml

Basically: 
- on new installations: RPM package created by default
- on upgrades: the script is called upon upgrade of `spacewalk-certs-tools` package, and it tries to create the package using the certificate looking for it in the default directory `/root/ssl-build`. If the location is not available or the user has changed the directory/certificate, the user MUST run the script manually by passing the full path of the certificate as an option.

aaannz commented:
Added info about new installation and upgrade and its relation to certificate rpm. Also added a note in trougleshooting about certificate rpm and link back to build host creation.

keichwa commented:
Next time, we'll consolidate the usage of btn: and menu: and other asoc constructs.
